io_scene_nif/modules/nif_export/property/texture/__init__.py

```diff
@@ -115,9 +166,12 @@ def determine_texture_types(self, b_mat):
         for b_texture_node in self.get_used_textslots(b_mat):
             NifLog.debug(f"Found node {b_texture_node.name} of type {b_texture_node.label}")
 
+            shown_label = b_texture_node.label
```

Comment:
Should pull this above the Niflog.debug, so that when it logs its uses `shown_label` variable to avoid the case its blank.
`NifLog.debug(f"Found node {b_texture_node.name} of type {shown_label}")`